src/pages/ProductDetail/ProductHeadSection.js

```diff
@@ -0,0 +1,225 @@
+import React from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const ProductHeadSection = ({ product }) => {
+  var settings = {
+    dots: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+  };
+  return (
+    <HeadSection>
+      <HeadImg className="wrapper">
+        <Slider {...settings} className="imgSlider">
+          {product.image &&
+            product.image.map(image => (
+              <div>
+                <div>
+                  <img alt="상품 사진" src={image} />
+                </div>
+              </div>
+            ))}
```

Comment:
```suggestion
           product.image.map((image, idx) => (
              <img key={idx} alt="상품 사진" src={image} />
            ))}
```
1. 불필요한 태그들은 삭제해주세요!
2. map 메서드 사용시에는 key prop을 꼭 부여해주세요!

Reply:
반영하였습니다!